content/millennium/r4/clinical/summary/procedure.md

```diff
@@ -52,21 +52,24 @@ Search for Procedures that meet supplied query parameters:
 
 ### Parameters
 
- Name              | Required?                      | Type          | Description
--------------------|--------------------------------|---------------|-----------------------------------------------------------------------
- `_id`             | This or `patient` or `subject` | [`token`]     | The logical resource id associated with the resource.
- `patient`         | This or `_id` or `subject`     | [`reference`] | Who the procedure is for. Example: `12345`
- `subject`         | This or `_id` or `patient`     | [`reference`] | Who the procedure is for. Example: `Patient/12345`
- `date`            | No                              | [`dateTime`]  | Date range into which the procedure falls. Example: `date=gt2015-09-24T12:00:00.000Z&date=le2020-07-15T16:00:00.000Z`
- `_revinclude`     | No                             | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target 
+ Name              | Required?     | Type          | Description
+-------------------|---------------|---------------|-----------------------------------------------------------------------
+ `_id`             | Conditionally | [`token`]     | The logical resource id associated with the resource. This parameter is required if `patient` or `subject` are not used.
+ `patient`         | Conditionally | [`reference`] | Who the Procedure is for. This parameter is required if `_id` or `subject` are not used. Example: `12345`
+ `subject`         | Conditionally | [`reference`] | Who the Procedure is for. This parameter is required if `_id` or `patient` are not used. Example: `Patient/12345`
```

Comment:
Wording on the parameters here should be used in DSTU2 resource's table as well.